### PR TITLE
Clarify the type of Localized URL

### PR DESCRIPTION
**Pull request by tdelmas**
### What is the issue and _why_ is it an issue?
The type `Localized String` is used at a multiple places with an additional specification that text must be of the type URL. It is an issue because it makes the specification more verbose, so less readable.

### Please describe some potential solutions you have considered (even if they aren’t related to GBFS).
I propose to define that `Localized String` where the text is a URL as `Localized URL`. The type in the header, so the type is more explicit.

But maybe the type `Localized URL` should be defined as `A fully qualified URL` to be even clearer?

image

### CODE REVIEW

**Comment by tdelmas** [comment]
Ping @richfab : Can I have your opinion on this change? Do you think it falls in the "Editorial changes" category?

**Comment by richfab** [comment]
Hi @tdelmas, sorry for the delay.
Could you please give more context as to what is the issue and _why_ it's an issue please?
Are you suggesting to replace the type `URL` by `Localized URL` in the spec?
Thank you in advance!

**Comment by tdelmas** [comment]
(sorry for the late reply too!)

- What is the issue: The type `Localized String` is used at a multiple place with an additional specification that `text` must be of the type URL. It is an issue because it makes the specification more verbose, so less readable.

I propose to define that "Localized String where the text is an URL" as "Localized URL" the type in the header, so the type is more explicit.

But maybe the type "Localized URL" should be defined as "A fully qualified URL" to be even clearer?

**Comment by richfab** [comment]
Tagging recent contributors to get your opinion: @mplsmitch, @testower, @cmonagle, @simonsolnes, @ezmckinn, @AntoineAugusti, @ArashMansouri, @jkurzanski.
Thank you!

File: gbfs.md [hunk 1]
```diff
@@ -207,14 +207,21 @@ Example: The `rental_methods` field contains values `creditcard`, `paypass`, etc
...
-* <a name="localized-string"></a> Localized String - A JSON element representing a String value that has been translated into a specific language.  The element consists of the following name-value pairs:
+* <a name="localized-string"></a> Localized String - A JSON element representing a String value that has been translated into a specific language. The element consists of the following name-value pairs:
 
   Field Name | REQUIRED | Type | Defines
   ---|---|---|---
   `text` | Yes | String | The translated text.
   `language` | Yes | Language | IETF BCP 47 language code.  Must match one of the values specified by the `languages` field in `system_information.json`.
 
   Most commonly specified as `Array<Localized String>` when specifying translations in multiple languages.  See [Localization](#localization) for more details.
+* <a name="localized-url"></a> Localized URL - A JSON element representing the URL of a ressource that has been translated into a specific language. The element consists of the following name-value pairs:
```
Comment (AntoineAugusti) [inline gbfs.md]:
```suggestion
* <a name="localized-url"></a> Localized URL - A JSON element representing the URL of a resource that has been translated into a specific language. The element consists of the following name-value pairs:
```

File: gbfs.md [hunk 2]
```diff
@@ -207,14 +207,21 @@ Example: The `rental_methods` field contains values `creditcard`, `paypass`, etc
...
+
+  Field Name | REQUIRED | Type | Defines
+  ---|---|---|---
+  `text` | Yes | URL | The URL of the translated ressource.
```
Comment (AntoineAugusti) [inline gbfs.md]:
```suggestion
  `text` | Yes | URL | The URL of the translated resource.
```

**Comment by testower** [comment]
I agree it is confusing. And made further confusing by the fact that it is not actually the URL itself that is localized (or translated), but the URL points the a version of the content that is localized according to the language code...

**Comment by AntoineAugusti** [comment]
👍 transport.data.gouv.fr, a simple and useful clarification.

**Comment by jkurzanski** [comment]
👍 for Vulog

**Comment by testower** [comment]
👍 from Entur as well

File: gbfs.md [hunk 3]
```diff
@@ -1254,6 +1260,7 @@ Field Name | REQUIRED | Type | Defines
 \-&nbsp;`station_ids` | OPTIONAL | Array | If this is an alert that affects one or more stations, include their ID(s). Otherwise omit this field. If both `station_ids` and `region_ids` are omitted, this alert affects the entire system.
 \-&nbsp;`region_ids` | OPTIONAL | Array | If this system has regions, and if this alert only affects certain regions, include their ID(s). Otherwise, omit this field. If both `station_ids` and `region_ids` are omitted, this alert affects the entire system.
 \-&nbsp; `url` <br/>*(as of v3.0-RC)* | OPTIONAL | Array&lt;Localized String&gt; | URL where the customer can learn more information about this alert.
+\-&nbsp; `url` <br/>*(as of v3.0-RC)* | OPTIONAL | Array&lt;Localized URL&gt; | A fully qualified URL where the customer can learn more information about this alert.
```
Comment (richfab) [inline gbfs.md]: Small typo (the original line was duplicated)

```suggestion
\-&nbsp; `url` <br/>*(as of v3.0-RC)* | OPTIONAL | Array&lt;Localized URL&gt; | A fully qualified URL where the customer can learn more information about this alert.
```

**Review by richfab — CHANGES_REQUESTED** [review]
One minor typo to fix. LGTM otherwise!

Thank you @tdelmas for this helpful contribution 💯

File: gbfs.md (hunk 3 above)
Comment (tdelmas) [reply gbfs.md]: Good catch, thanks!

**Review by richfab — APPROVED** [review]
LGTM!
Merging now 🚀

**Pull request merged**